frontend/lang/en.json

```diff
@@ -33,15 +53,49 @@
     },
     "StandardFooter": {
         "A project of": "A project of",
+        "File a Bug": "File a Bug",
         "Terms of Use": "Terms of Use",
         "Privacy": "Privacy",
-        "About": "About"
+        "About": "About",
+        "Manage Fake Users": "Manage Fake Users",
+        "Toggle Debug Info": "Toggle Debug Info",
+        "Dev Tools": "Dev Tools"
     },
     "StandardNav": {
         "Home": "Home",
-        "File a Bug": "File a Bug",
         "Admin": "Admin",
+        "My Stuff": "My Stuff",
         "Sign Out": "Sign Out",
         "Sign In": "Sign In"
-    }
+    },
+    "UserEditor": {
+        "The name that will be associated with": "The name that will be associated with ",
+        "you": "your public activities on the platform",
+        "this user": "this user's public activities on the platform",
+        "may be public": "Note that this name may be accessible to a public audience.",
+        "language help text": "What language should platform freatures that support internationalization default to for this user?",
+        "admin help text": "If enabled, this user will have administrator priveledges.",
+        "is admin": "Is an Administrator",
+        "is not admin": "Is not an Administrator",
+        "super admin help text": "If enabled, this user will have super-administrator priveledges.",
+        "is super admin": "Is a Super Administrator",
+        "is not super admin": "Is not a Super Administrator"
+    },
+    "UserToolbar": {
+        "Profile": "Profile",
+        "Edit": "Edit"
+    },
+    "initiative/invitations": {
+        "Heading": "Initaitive Invitations",
+        "No Invitations": "This initiative does not require an invitation to join - anyone who finds this initiative can join it.",
+        "Yes Invitations": "This initiative requires an invitation (a unique, one-time code) to join. You can create these invitations below.",
+        "You can change": "You can change this behavior on the",
+        "edit initiative page": "edit initiative page",
+        "Create Initiative Invitations": "Create Initiative Invitations",
+        "New Invitations": "New Invitations",
+        "New Invitations Help": "Create new invitations by clicking the button below. You can create as many invitations as you want, but each invitation can only be used once. Invitation codes must only contain alphanumeric characters (a-z, A-Z, 0-9), underscores and dashes. You can create multiple invitations at once by separating them with tabs, commas, newlines, or spaces.",
+        "Create": "Create",
+        "Unused": "Unused",
+        "Includes Illegal Characters": "Includes illegal characters"
+    },
```

Comment:
General question about i18n: does the vue/nuxt-i18n stuff allow you to provide context for translations? I can see that being useful for some of these, where without knowing how/where/when we're using it, it might be harder to translate well.